config/routes.rb

```diff
@@ -1,5 +1,18 @@
 Rails.application.routes.draw do
-  root 'homes#index'
   devise_for :users
+  
+  root 'homes#index'
+
+  get "/movies", to: "homes#index"
+
+  get "/movies/:id", to: "homes#index"
+
+
```

Comment:
There dont need to be any empty lines between the routes from 4-8.  Also an extra empty line on 10